codewiki/src/be/llm_services.py

```
@@ -32,6 +32,25 @@ def get_llm_cache():
+    Args:
+        config: Configuration object containing cache_size in analysis_options
+    """
+    try:
+        from codewiki.src.be.caching import get_llm_cache
+
+        cache_size = config.analysis_options.cache_size
+        llm_cache = get_llm_cache()
+        llm_cache.max_size = cache_size
+
+        logger.info(f"LLM cache initialized with max_size={cache_size}")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)